test/integration/Identity.test.ts

```diff
@@ -320,6 +328,43 @@ describe('A Solid server with IDP', (): void => {
       res = await state.session.fetch(newWebId, patchOptions);
       expect(res.status).toBe(205);
     });
+
+    it('always has control over data in the pod.', async(): Promise<void> => {
+      const podBaseUrl = `${baseUrl}${podName}/`;
+      const brokenAcl = '<#authorization> a <http://www.w3.org/ns/auth/acl#Authorization> .';
+
+      // Make the acl file unusable
+      let res = await state.session.fetch(`${podBaseUrl}.acl`, {
+        method: 'PUT',
+        headers: { 'content-type': 'text/turtle' },
+        body: brokenAcl,
+      });
+      expect(res.status).toBe(205);
+
+      // Owner locked is locked out
```

Comment:
?

Reply:
As in, can't parse the sentence.

But also, is that correct behavior? Can we point to the spec requirement?

Reply:
Well that's just a bad sentence, not sure what happened there. W

hat happens here is that the acl file has been updated so that they don't have access to the resource anymore (all modes have been removed), which is why they get a 403 when accessing the resource. So the spec requirement is the WebACL spec. Due to the new permission reader they can update the acl resource though.